ci-operator/config/terraform-redhat/terraform-provider-rhcs/terraform-redhat-terraform-provider-rhcs-main.yaml

```diff
@@ -392,6 +436,8 @@ tests:
       CLUSTER_PROFILE: rosa-hcp-pl
       QE_USAGE: rosa-hcp-private-optional-presubmit
       REGION: us-west-2
+      RHCS_CLUSTER_NAME_PREFIX: ci
```

Comment:
If `RHCS_CLUSTER_NAME_PREFIX` is always `ci` why not hardcode it in the step to reduce variable use? Besides, for you have defined `QE_USAGE`, maybe `RHCS_CLUSTER_NAME_SUFFIX` also can be defined in the step by it.